documents/src/pages/index.js

```diff
@@ -12,7 +13,11 @@ function HomepageHeader() {
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
+        <h1 className="hero__title">
+          <Translate
+            id="homepage.title"
+            description="homepage title"></Translate>
```

Comment:
フレキシブルな切り替えは非対応ですが，例えばdefaultLocale を `en` とすると正しく表示されません．したがって，以下の対応が必要です．

```suggestion
          <Translate
            id="homepage.title"
            description="homepage title">{siteConfig.title}</Translate>
```

Reply:
修正・手元で確認しました！
https://github.com/FairyDevicesRD/thinklet.app.developer/pull/1/commits/5d2c32567f7d7f0695124ad25cb26b7a5ea9df12